crates/rari/src/rsc/rendering/streaming/js/streaming_init.js

```
@@ -186,7 +18,10 @@ if (!globalThis['~suspense']) {
     discoveredBoundaries: [],
     pendingPromises: [],
     currentBoundaryId: null,
+    renderGeneration: globalThis['~rsc'].renderGeneration,
   }
+} else {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)